subprojects/distributions/src/changes/changes-since-4.0.txt

```diff
@@ -0,0 +1,5 @@
+Implemented interface org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Method org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Class org.gradle.api.invocation.Gradle;All regressions accepted
```

Comment:
You only add here regressions that were not automatically tagged. In this case, all members of the `IncludedBuild` class were incubating, but still, there are things not caught automatically by the rule (the plugin typically doesn't allow to post-process all violations for a class, to tell that if all members were accepted, then we can also accept the class).